functions/helpers.ts

```diff
@@ -52,3 +58,146 @@ export function getBaseUrl(isSandbox: boolean): string {
   }
   return "https://giftcards-sandbox.reloadly.com";
 }
+
+export async function findBestCard(countryCode: string, amount: BigNumberish, accessToken: AccessToken): Promise<GiftCard> {
+  const supportedCountry = allowedCountries.find((listItem) => listItem.code == countryCode);
```

Comment:
Have you considered exporting `Map`s for `allowedCountries` and the skus instead of arrays and replacing `find()` usage?

Reply:
I was hoping I would access the array with different keys but I see I haven't. So Map sounds like a good idea. I will implement it.

Reply:
I have refactored the countries array to Record instead of Map. I have found it easier to fill them as compared to Map. However, I have not changed the SKU array. The problem is that we search skus array for different fields, country code, SKU. Also, we load gift cards list from API that also comes in the form of an array. To me, it looked better to keep them as array.